storm-core/src/clj/backtype/storm/ui/core.clj

```diff
@@ -1108,12 +1108,8 @@
   (GET "/api/v1/topology/:id/profiling/start/:host-port/:timeout"
        [:as {:keys [servlet-request]} id host-port timeout & m]
        (thrift/with-configured-nimbus-connection nimbus
```

Comment:
These all seem to be missing `(populate-context! servlet-request)` We need to add it as the first thing in each of the REST requests